README.rst

```diff
@@ -21,6 +21,14 @@ zyte-spider-templates
 
 Spider templates for automatic crawlers.
 
+This library contains Scrapy_ spiders that are intended to be used with the
+Zyte features such as `Zyte API`_. There is a `sample Scrapy project`_ for this
```

Comment:
This library can be used without any Zyte features.

Reply:
Or, rather, how to best describe this library in one sentence.

Reply:
How about this?